### DISCUSSION TITLE
Inputted the wrong Client ID HELP

### DISCUSSION THREAD

**Discussion by MMikolaj1** [ekutner/home-connect-hass#441] [Community Support]
When setting up I inputted the wrong client ID and secret key and takes me over to an error client id page. When I try to add the integration again it keeps using the wrong client id and doesn't give ma chance to change it. I've tried uninstalling the integration via hacs and reinstalling but getting the same issue every time. Once you enter the id it just seems to remember it.

**Comment by ekutner** [comment]
It's actually the first bullet in the FAQ, follow the instructions there
